iib/workers/tasks/build.py

```diff
@@ -584,16 +586,14 @@ def handle_add_request(
 
     set_request_state(request_id, 'in_progress', 'Creating the manifest list')
     output_pull_spec = _create_and_push_manifest_list(request_id, arches)
-
+    modified_from_index = _check_to_modify_index_image(
```

Comment:
I think `from_index` terminology shouldn't be used here. This is the pull spec of the `index_image` built by IIB. Consider `modified_index_image`.